scripts/cmake/FindIbex.cmake

```diff
@@ -13,6 +13,9 @@ if(DOWNLOAD_DEPENDENCIES AND (NOT DEFINED IBEX_DIR))
 
    message("[ibex] Setting up ibex")
    if(ACADEMIC_BUILD)
+      message(WARNING "ibex is using soplex. Soplex is a tierce library under ZIB licence, not LGPL. \
+      By configuring Ibex with Soplex, you implicitely accept the terms of this licence. \
```

Comment:
Please rephrase to something similar as https://github.com/esbmc/esbmc/blob/589b1880df82014034c249ad6a9ede9928173ed6/src/solvers/yices/CMakeLists.txt#L80

Most importantly, this affects redistributability of the binary: it cannot be distributed anymore. Also users cannot implicitly accept license terms.

Reply:
Changed the message to: 
message(WARNING "the version of ibex you have is ZIB licensed, distribution is impossible.")

Although, I didn't get the lost in space reference here. 
Should i change it to STATUS?

Reply:
Warning is fine, many thanks!

If you haven't watched it, I'd recommend the 2018 reboot of the series. ;)